photutils/centroids/core.py

```diff
@@ -479,3 +491,100 @@ def centroid_sources(data, xpos, ypos, box_size=11, footprint=None,
         ycentroids.append(ycen + slices_large[0].start)
 
     return np.array(xcentroids), np.array(ycentroids)
+
+
+def centroid_epsf(data, mask=None, oversampling=4, shift_val=0.5):
+    """
+    Calculates centering shift of data using pixel symmetry, as described by Anderson
+    and King (2000; PASP 112, 1360).
```

Comment:
```suggestion
    and King (2000; PASP 112, 1360) in their ePSF-fitting algorithm.
```
this suggested addition is to clarify to the uninformed reader why the function is called `centroid_epsf`